entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Just wondering about this file, is it supposed to have executable permissions? I built the Dockerfile image on my computer and when I tried to run it I get an error involving `entrypoint.sh`. I had to run `chmod +x entrypoint.sh`, rebuild the image, and rerun to get the image to run.

Reply:
Good point! I'll change that.